src/compiler/_namespaces/ts.ts

```diff
@@ -10,6 +10,7 @@ export * from "../tracing";
 export * from "../types";
 export * from "../sys";
 export * from "../path";
+/** @knipignore */

```

Comment:
Is this only needed because the file isn't present before running knip? Probably we should add a `knip` task to `Herebyfile.mjs` and have it depend on `generateDiagnostics`.

A task there is also more in line with how we set things up; we rarely use any npm scripts.

Reply:
Indeed, OK